include/z64animation.h

```diff
@@ -24,37 +24,44 @@ typedef struct {
     /* 0x007 */ u8 nextLimbIndex;   // The parent limb's next limb index into the limb table.
     /* 0x008 */ Gfx* displayLists[1]; // Display lists for the limb. Index 0 is the normal display list, index 1 is the
                                       // far model display list.
-} SkelLimbEntry;                      // Size = 0xC or 0x10
+} StandardLimb;                      // Size = 0xC or 0x10
 
+#if 0
 typedef struct {
-    /* 0x000 */ SkelLimbEntry* limbs[1]; // One or more limbs, index 0 is the root limb.
+    /* 0x000 */ StandardLimb* limbs; // One or more limbs, index 0 is the root limb.
 } Skeleton;                              // Size >= 4
+#endif
```

Comment:
Should this be deleted entirely instead of commented out?

Reply:
whoops, I added this for testing and forgot to remove it. Good catch!